argus/config/__init__.py

```diff
@@ -25,8 +25,8 @@
     option_class(CONFIG).register()
 
 _DEFAULT_CONFIG_FILES = [
-    config_file for config_file in ("argus.conf", "etc/argus/argus.conf",
-                                    "/etc/argus/argus.conf")
+    config_file for config_file in ("/etc/argus/argus.conf",
```

Comment:
We want the `argus.conf` to have the highest priority, how they are set up right now the `/etc/argus/argus.conf` will overwrite fields from `argus.conf`. If let's say we have a random `image_ref` set in `/etc/argus/argus.conf` the option set in `argus.conf` is useless.